IES Specification Docs/Information Exchange Standard r4.2.0.n3

```diff
@@ -1750,13 +1671,17 @@ ies:ISO3166_1Alpha_3 rdf:type rdfs:Class .
 ies:ISO3166_1Alpha_3 sparx:guid "{B92D79E2-9E7D-4df7-8D38-3D884AA09AD2}" . 
 ies:ISO3166_1Alpha_3 rdfs:comment "ISO 3166-1 alpha 3 (3-Letter Country Code)" . 
 ies:ISO3166_1Alpha_3 rdfs:subClassOf ies:GeoIdentity . 
+ies:ISO4217Code rdf:type rdfs:Class . 
+ies:ISO4217Code sparx:guid "{598ACBB6-DF51-4bd9-A5DD-52EDE1895327}" . 
+ies:ISO4217Code rdfs:comment "ISO4217 three-letter code (e.g. USD, GBP, EUR, etc.)" . 
+ies:ISO4217Code rdfs:subClassOf ies:Identifier . 
 ies:ISO639-3Code rdf:type rdfs:Class . 
 ies:ISO639-3Code sparx:guid "{ECFED94D-CC69-46b9-B09D-B282D5665787}" . 
 ies:ISO639-3Code rdfs:comment "ISO639-3 three-letter language code" . 
 ies:ISO639-3Code rdfs:subClassOf ies:Identifier . 
 ies:iso8601PeriodRepresentation rdf:type owl:DatatypeProperty . 
 ies:iso8601PeriodRepresentation sparx:guid "{E9372543-434E-45d3-A1F0-8D711952D10A}" . 
-ies:iso8601PeriodRepresentation rdfs:comment "A ISO8601 datetime (as xsd:dateTime if possible) that represents the ParticularPeriod.Note: the representation is also encoded in the URI of the period, this is an additional required attribute to enable querying by dateTime and SPARQL temporal operations." . 
+ies:iso8601PeriodRepresentation rdfs:comment "A ISO8601 datetime (as xsd:dateTime) that represents the ParticularPeriod. This representation is also encoded in the URI of the period, this is an additional required attribute to enable querying by dateTime and SPARQL temporal operations. The literal string shall be encoded in UTC (Coordinated Universal Time) but unlike the URI, it must be punctuated. For example: 2007-01-18T15:30:00Note: the representation is also encoded in the URI of the period, this is an additional required attribute to enable querying by dateTime and SPARQL temporal operations." . 
```

Comment:
A bit of accidental duplication in here --Lisa

Reply:
resolved in https://github.com/dstl/IES4/pull/9/commits/7ee474edf985ac5e0a665347fb922271e6675f21